### PR TITLE
fctrFraxBP-fcGLP

### PR DESCRIPTION
**Pull request by johnnyonline**

### CODE REVIEW

**Comment by johnnyonline** [comment]
Hey guys. Ill merge this in a couple of hours, then deploy this + fctr3Crypto-fcGlp to Arbi mainnet. Lmk of any feedback

File: script/arbitrum/utils/concentrators/curve/InitFraxBPGlp.sol [hunk 1]
```diff
@@ -1,123 +1,137 @@
...
        
-//         _initSwapFraxBPGlp(_fortressSwap);
+        _initSwapFraxBPGlp(_fortressSwap);
         
-//         // ------------------------- init TriCrypto compounder -------------------------
+        // ------------------------- init TriCrypto compounder -------------------------
 
-//         uint256 _convexPid = 5;
-//         uint256 _poolType = 1; 
+        uint256 _convexPid = 10;
```
Comment (xh3b4sd) [inline script/arbitrum/utils/concentrators/curve/InitFraxBPGlp.sol:18]: Can you explain what the convex pid here is and why it changed from 5 to 10?

**Comment by ashablovskiy** [comment]
No comments here

File: script/arbitrum/utils/concentrators/curve/InitFraxBPGlp.sol (hunk 1 above)
Comment (johnnyonline) [reply script/arbitrum/utils/concentrators/curve/InitFraxBPGlp.sol:18]: It's ID of Curve LP token in Convex stacking contract. It was updated by Convex a few weeks ago.

Comment (xh3b4sd) [reply script/arbitrum/utils/concentrators/curve/InitFraxBPGlp.sol:18]: Say we run our vaults and they change the ID again. How do we notice and what will happen until we fix it on our side?

**Pull request merged**